<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportNumberResidentsAddressSetup</class>
 <widget class="QDialog" name="ReportNumberResidentsAddressSetup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="1" column="1" colspan="3">
    <widget class="QFrame" name="frmAge">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbAttachType"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Дата прикрепления</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>129</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblArea">
     <property name="text">
      <string>ЛПУ прикрепления</string>
     </property>
     <property name="buddy">
      <cstring>cmbArea</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>По&amp;л</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QComboBox" name="cmbAttachCategory">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Постоянное</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Временное</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Выбыл</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>141</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblAttachType">
     <property name="text">
      <string>Прикрепление категория</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Тип прикрепления</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="CDateEdit" name="edtAttachBegDate">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>по</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtAttachEndDate">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="CPolyclinicComboBox" name="cmbArea"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cmbSex">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Во&amp;зраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblAddressType">
     <property name="text">
      <string>Тип адреса</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="cmbAddressType">
     <item>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проживание</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CPolyclinicComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbSex</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>cmbArea</tabstop>
  <tabstop>cmbAttachCategory</tabstop>
  <tabstop>edtAttachBegDate</tabstop>
  <tabstop>edtAttachEndDate</tabstop>
  <tabstop>cmbAttachType</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportNumberResidentsAddressSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportNumberResidentsAddressSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
